library/Fission/URL/DomainName/Types.hs

```diff
@@ -5,7 +5,7 @@ import qualified RIO.Char            as Char
 import qualified RIO.Text            as Text
 
 import           Database.Persist.Postgresql hiding (get)
-import           Data.Swagger                hiding (get)
+import           Data.Swagger                hiding (get, host)
```

Comment:
was getting a compiler warning from this. not technically part of this PR but I wanted it to go away so :man_shrugging:

Reply:
Weird